amundsen_application/api/search/v0.py

```diff
@@ -84,12 +84,13 @@ def search_table_qs() -> Response:
     valid_categories = ['column', 'database', 'schema', 'table', 'tag']
     for category in valid_categories:
         values = filters.get(category)
-        if type(values) == str and len(values) > 0:
-            filter_payload[category] = [values, ]
-        if type(values) == dict:
-            value_list = list(values.keys())
-            if len(value_list) > 0:
-                filter_payload[category] = value_list
+        if values is not None:
+            if type(values) == str:
+                filter_payload[category] = [values, ]
+            if type(values) == dict:
+                value_list = list(values.keys())
```

Comment:
Will take a look and I will update this to be more robust. This logic right now is "safe", but its only safe because I am leveraging my knowledge of what I've written in the Redux logic, `hive: false` would never exist because I remove it from the dictionary but I will update to not rely on that being the case.

Reply:
Added some cleanup in next commit.